guides/common/assembly_using-pxe-to-provision-hosts.adoc

```diff
@@ -4,10 +4,16 @@ include::modules/con_prerequisites-for-bare-metal-provisioning.adoc[leveloffset=
 
 include::modules/proc_configuring-the-security-token-validity-duration.adoc[leveloffset=+1]
 
+:creating-hosts-with-unattended-provisioning:
```

Comment:
maybe introduce "pxe-provisioning" attribute similar to proxmox/vmware?

Reply:
Isn't PXE common to all bare metal (with PXE) and VM provisioning?
In other words, I don't think that this assembly follows the same principle as proxmox/vmware...
I wouldn't introduce an attribute for this.